### PR TITLE
Add cross-layer energy paper

### PR DESCRIPTION
**Pull request by jaywonchung**

### CODE REVIEW

**Comment by jaywonchung** [comment]
@mosharaf The arXiv replacement submission was released from hold. I think it'll be out in two days.

The problem is that we actually had a separate entry on our pub list pointing to the arXiv entry. When the arXiv replacement submission gets announced, the two will go out of sync (both abstract and title).

I think we can (1) update the older entry on our homepage pub list with the newer abstract and title, and (2) keep the changes proposed by this PR, creating a new pub list entry containing the final PDF file.

WDYT?

**Comment by mosharaf** [comment]
Historically, we never went back to update an arXiv enrty in bib file even though a newer version has replaced it.

**Comment by jaywonchung** [comment]
In that case, one way is to just implement (2) above. Another way is to withdraw the arXiv replacement submission and implement (2).

**Comment by jaywonchung** [comment]
The downside of the former is that we need to let Nishil know that we're not changing the arXiv entry

**Comment by mosharaf** [comment]
You mean the latter, no? Former seems the best course of action: keep old arXiv entry in our website as is; update arXiv as you've had; create new entry => just merge this PR.

**Comment by jaywonchung** [comment]
Ah yeah I meant the latter. Going with the former. The PR is ready for your review and once it's merged, there's nothing more to do other than me keeping track of the arXiv replacement submission.

File: source/_data/SymbioticLab.bib [hunk 1]
```diff
@@ -1739,6 +1739,20 @@ @Article{crosslayer-energy:arxiv24
   }
 }
 
+@inproceedings{crosslayer-energy:eecs24,
+  author        = {Jae-Won Chung and Nishil Talati and Mosharaf Chowdhury},
+  booktitle     = {Energy-Efficient Computing for Science Workshop},
+  title         = {Toward Cross-Layer Energy Optimizations in AI Systems},
```
Comment (mosharaf) [inline source/_data/SymbioticLab.bib:1745]: {AI}

**Review by mosharaf — CHANGES_REQUESTED** [review]

**Review by mosharaf — APPROVED** [review]

**Pull request merged**